Week-01/Day-01_Convert-Ages-To-Days/convert_ages_to_days_botiboti/src/main.rs

```diff
@@ -1,5 +1,10 @@
 use std::io;
 
+fn calc(str: String) -> i32 {
+    let days: i32 = str.trim().parse().expect("Not an integer.");
```

Comment:
This can crash. It's a function that turns years into days, it should not be possible to crash it.

Reply:
parsing separated, stays in main

Reply:
:+1: